docs/API.md

```diff
@@ -1713,23 +1733,15 @@ type ResourcesRevision struct {
 }
 ```
 
-#### GET *id*/resources/name.stream[-revision]/arch/filename
+#### GET *id*/resources/name[/revision]
 
-Getting from the `/resources` path retrieves a charm resource from the charm
-with the given id. If version is not specified, it retrieves the latest version
-of the resource. The SHA-256 hash of the data is specified in the HTTP response
-headers.
+Getting from the `/resources` path retrieves a charm resource from the charm with the given id. If version is not specified, it retrieves the latest version of the resource. The SHA-256 hash of the data is specified in the HTTP response headers.
```

Comment:
Is SHA-256 still what we are using?  I thought we were moving to SHA-384 for resources.